editor/project.clj

```diff
@@ -247,6 +247,7 @@
                                 :source-paths      ["src/dev"]
                                 :repl-options      {:init-ns user}
                                 :proto-paths       ["test/proto"]
+                                :java-source-paths ["test-java"]
```

Comment:
I wanted a Java class with the new `ConsistentHashCode` attribute for the editor tests, but this is the first time we'd wanted to include a Java class in our tests. Ideally, we'd probably like to split test source roots into `test/clj` and `test/java` like we do for `src`, but I didn't want to change the directory structure in this PR.